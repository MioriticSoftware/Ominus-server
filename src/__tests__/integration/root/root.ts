import { describe, expect, test } from '@jest/globals';
import request from 'supertest';

import app from '../../../server';

describe('Test example', () => {
    test('GET /', (done) => {
        request(app)
            .get('/')
            .expect(200)
            .expect((res) => {
                expect(res.text).toMatchSnapshot();
            })
            .end((err) => {
                if (err) return done(err);
                return done();
            });
    });
});
