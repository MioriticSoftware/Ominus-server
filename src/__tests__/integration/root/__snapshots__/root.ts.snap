// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test example GET / 1`] = `"Hello World"`;
